code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -245,6 +245,21 @@
 		current_type = initial(current_type.parent_type)
 	return current_type
 
+///returns a list of strains(xeno castedatum paths) that this caste can currently evolve to
+/datum/xeno_caste/proc/get_strain_options()
+	var/datum/xeno_caste/root_type = type
+	while(initial(root_type.parent_type) != /datum/xeno_caste)
+		root_type = initial(root_type.parent_type)
+	var/list/candidates = typesof(root_type)
```

Comment:
Why do you have to do this ?

Reply:
We get the child of xeno_caste (e.g runner)  then get all children of that to find the strains

Reply:
Why not a Glob list somewhere ? less snowflaky

Reply:
How so? I need to return the strains for a given type, and I dont want to force people to add defines to everything for a glob lookup

Reply:
Can't you just have a GLOBAL_LIST_INIT to do this?

Reply:
Done